config.sample.json

```diff
@@ -1,6 +1,10 @@
 {
     "default_hs_url": "https://matrix.org",
     "default_is_url": "https://vector.im",
+    "disable_custom_urls": false,
+    "disable_guests": false,
+    "disable_login_language_selector": false,
+    "disable_3pid_login": false,
```

Comment:
Note to self: These should be pulled into GranularSettings when it lands

Reply:
decided to not make these granular - they are app-level anyways and affect people who aren't logged in, so granularity isn't feasible.